src/Microsoft.Azure.WebJobs.Host/Bindings/BindingProviders/AsyncCollectorBindingProvider.cs

```diff
@@ -0,0 +1,303 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+
+namespace Microsoft.Azure.WebJobs.Host.Bindings
+{
+    // General rule for binding parameters to an AsyncCollector. 
+    // Supports the various flavors like IAsyncCollector, ICollector, out T, out T[]. 
+    internal class AsyncCollectorBindingProvider<TAttribute, TType> : FluidBindingProvider<TAttribute>, IBindingProvider
```

Comment:
It should be "Fluent", referring to the fluent design pattern for APIs. ([https://en.wikipedia.org/wiki/Fluent_interface](https://en.wikipedia.org/wiki/Fluent_interface))  . 


---
In reply to: [99674533](https://github.com/Azure/azure-webjobs-sdk/pull/1006#discussion_r99674533) [](ancestors = 99674533)